### ISSUE TITLE
Opus: "--downmix-stereo" is ignored in 2.44

### ISSUE THREAD

**Issue by stefan-i** [staxrip/staxrip#1569]
**Describe the bug**
When setting downmix to "stereo" in audio settings, the resulting "--downmix-stereo" in the script is ignored when encoding. It worked fine in 2.42.2, so it seems like a new bug.

**Expected behavior**
stereo opus audio tracks, not multichannel like in the source

**How to reproduce the issue**
Happens with any source file that conatins multichannel audio. Tested on two different computers, so not tied to a specific environment.

**Provide information**
Logfile: https://pastebin.com/SzvrkSHC
I am 100% certain I set the --downmix-stereo option with this and other encodes. When I click on "show commandline" in the audio settings it is there, but it seems to disappear in the actual encode as can be seen in the logfile. 
By the way, I did _not_ activate the "center/speech optimized Stereo" option, but I noticed that when it is activated the commandline parts that are created when using the "downmix" menu also disappear, so maybe the bug is somehow connected to this new option.

Thank you for looking into this. If you need further info, please let me know.
Kind regards
Stefan

**Comment by Dendraspis** [comment]
I definitely can confirm that.  Let's see if the next release will fix it without creating new issue regarding normalizing...  🙄

**Comment by Dendraspis** [comment]
Can you confirm, that this issue is solved for you in `v2.44.1` ?

**Comment by stefan-i** [comment]
> Can you confirm, that this issue is solved for you in `v2.44.1` ?
Yes, seems to work fine now. Encodes are in stereo now and sound fine.
Thank you for fixing this so quickly!

**Comment by Dendraspis** [comment]
Thanks for the feedback.  😊 

You're welcome, couldn't accept such an annoying bug in this release.  😅
